<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerParamsPreviewDialog</class>
 <widget class="QDialog" name="PrimerParamsPreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Amplicon Bounds</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>76</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Forward Prefix</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Amplicon Length</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Reverse Suffix</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>T&lt;sub&gt;m&lt;/sub&gt; Range</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>120</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Forward Suffix</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Primer Length</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Reverse Prefix</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ampliconBoundsValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>0</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ampliconLengthValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="primerLengthValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="tmRangeValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="forwardPrefifxValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>80</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="reversePrefixValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>100</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="forwardSuffixValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>120</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="reverseSuffixValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>140</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="sodiumConcentrationValue">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>160</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLabel" name="sodiumConcentrationLevel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 1px solid #EEE;</string>
         </property>
         <property name="text">
          <string>Sodium Levels</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCommandLinkButton" name="commandLinkButton">
           <property name="text">
            <string>Create more primers</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>PrimerParamsPreviewDialog</receiver>
   <slot>onCreateMoreClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCreateMoreClicked()</slot>
 </slots>
</ui>
